src/libostree/ostree-repo-prune.c

```diff
@@ -299,26 +307,19 @@ repo_prune_internal (OstreeRepo        *self,
   return TRUE;
 }
 
-/**
- * ostree_repo_traverse_reachable_refs:
- * @self: Repo
- * @depth: Depth of traversal
- * @reachable: (element-type GVariant GVariant): Set of reachable objects (will be modified)
- * @cancellable: Cancellable
- * @error: Error
- *
- * Add all commit objects directly reachable via a ref to @reachable.
- *
- * Locking: shared
- * Since: 2018.6
- */
-gboolean
-ostree_repo_traverse_reachable_refs (OstreeRepo *self,
-                                     guint       depth,
-                                     GHashTable *reachable,
-                                     GCancellable *cancellable,
-                                     GError      **error)
+static gboolean
+ostree_repo_traverse_reachable_refs_with_options (OstreeRepo                    *self,
```

Comment:
Also while you're here, mind dropping the `ostree_` from the function prefix?  In new code I've been trying to reserve that for actual public functions.  Further a common pattern is to add `_internal` or `_impl` to the function.

So this could be just e.g. `traverse_reachable_internal()` or perhaps just `traverse_impl` or so.